### DISCUSSION TITLE
Welcome to Oracle Linux Discussions!

### DISCUSSION THREAD

**Discussion by totalamateurhour** [oracle/oracle-linux#35] [General Oracle Linux questions]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by pthukuntla** [comment]
does anyone has compatibilty issues with x2go client on oracle Linux 8. We are recently migrated to oracle linux 8.9, from centos. 
x2go client was working absolutely fine in centos but somwhow having rendering issues and slowness in oracle linux, pls suggest if there is any solution or any other remote toll other than VNC

**Comment by honglinsu** [comment]
are you using x2client from the Oracle Linux yum channel (ol8_developer_EPEL)?

**Reply by pthukuntla** [reply]
yes

**Comment by honglinsu** [comment]
There are several options such as remmina, xorgxrdp, xrdp, and xrdp-selinux that are known to work with Oracle Linux and other Linux distros. Here are a couple useful links. 

- https://www.techtarget.com/searchVirtualDesktop/tip/How-to-deploy-and-configure-xrdp-on-Linux
- https://www.nomachine.com/

**Reply by pthukuntla** [reply]
Thanks for the suggestion, xrdp looks better option.
